Assets/Fungus/Scripts/Components/Flowchart.cs

```diff
@@ -319,18 +316,13 @@ public static void BroadcastFungusMessage(string messageName)
         public virtual float Zoom { get { return zoom; } set { zoom = value; } }
 
         /// <summary>
-        /// Scrollable area for Flowchart editor window.
-        /// </summary>
-        public virtual Rect ScrollViewRect { get { return scrollViewRect; } set { scrollViewRect = value; } }
-
-        /// <summary>
         /// Current actively selected block in the Flowchart editor.
         /// </summary>
         public virtual Block SelectedBlock
         { 
             get
             {
-                return selectedBlocks.FirstOrDefault();
+                return selectedBlocks.Count > 0 ? selectedBlocks[0] : null;
```

Comment:
I believe I did this because the profiler said `FirstOrDefault()` had a > 0 CG Alloc. I have no idea why it would, though. I'll double check.